data/sql_updates/totals_candidate_committee_presidential.sql

```diff
@@ -210,7 +217,7 @@ with last_cycle as (
         ), election_totals_with_begginning_and_ending_totals as(
             select
                 election_totals.*,
-                begginning_and_ending_totals.coverage_start_date,
+                --begginning_and_ending_totals.coverage_start_date,
```

Comment:
Yeah I was just making sure nothing was left over inadvertently or meant to be uncommented.  Thanks!